rockcraft/lifecycle.py

```diff
@@ -26,8 +27,38 @@
 from .parts import PartsLifecycle, Step
 from .project import Project, load_project
 from .providers import capture_logs_from_instance
+from rockcraft.errors import RockcraftInitError
 
 
+def init(rockcraft_yaml_content: str) -> None:
+    """Initialize a rockcraft project.
+
+    Args:
```

Comment:
note that we use rst for our documentation

Reply:
good catch...bad autodoc setup on the IDE. Fixed